tests/blobstore_common_tests.py

```diff
@@ -130,6 +141,20 @@ def testListV2(self):
         ))
         self.assertEqual(len(items), 1)
 
+        # test verbose flag
+        items = list(self.handle.list_v2(
+            self.test_fixtures_bucket,
+            "testList/prefix",
+            verbose=True
+        ))
+        self.assertEqual(len(items), 10)
+        for i, item in enumerate(items):
```

Comment:
would prefer: 

`for ix, (key, metadata) in enumerate(items):`

i'm like 99% sure that works. :)